engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/UpdateByWindowTime.java

```diff
@@ -269,7 +273,7 @@ public void computeAffectedRowsAndOperators(UpdateByWindowBucketContext context,
                 modifiedAffectedPrev.retain(ctx.timestampValidRowSet);
                 tmpAffected.insert(modifiedAffectedPrev);
             }
-            // naturally need to compute all modified rows
+            // compute all modified rows
             tmpAffected.insert(upstream.modified());
```

Comment:
Remind me why we need this. Don't we just care about the rows *affected* by mods, rather than the mods themselves? Maybe these are redundant.

Reply:
Short answer, we care if timestamps have changed because a modified ts will change the row window.  Have added ts modified flag to inform this choice.